README.md

```diff
@@ -181,6 +182,10 @@ To run the unit tests:
 npm run test
 ```
 
+## Known Limitations
+
+- Versioned repository support : This plugin does not support the use of versioned repositories.
+- Maximum file size allowed : Attachments are limited to a maximum size of 100 MB.
```

Comment:
For headers you can just keep Repository and File size. e.g
Repository: <Description>
File size: <Description>